### PR TITLE
First notebook tutorial

### PR DESCRIPTION
**Pull request by ivalaginja**
Adresses #70 
I am creating a notebook that covers the configfile setup in detail, fits a simple transit, calculates a set of limb darkening coefficients, provides a simplified example of the whole marginalisation procedure and has some final notes about how to run the main script.

### CODE REVIEW

**Comment by ivalaginja** [comment]
@hrwakeford I added the simplified marginalisation example to the intro tutorial and this is ready for review now.

**Comment by ivalaginja** [comment]
As it turns out, the false plot at the end of the notebook was wrong because it was plotting the initial model before plotting it as opposed to plotting a fit model. This was due to the fact that we reset the transit model **after** each fit, which means that once the loop of all fits is complete, the model got reinitialized.

I have moved the model reset to the top of the fitting loop (which makes more sense anyway, as you want to make sure your model is reset to the input parameters **before** you perform the fit) and I opened a new issue #88 to evaluate whether we want to move this reset to the top of the fitting also in our main script `marginalisation.py`. I think the answer is yes, but in that script we access all fit parameters through the diagnostics arrays rather than the model directly, so it's not an issue (yet?).

As for the very last section of the notebook, this was intended to be a primer and general instructions on how to run the full code. I don't think any specific examples make much sense in that part, as we covered those in the initial parts of the notebook. I can see though how this last bit looks very sad and lonely and confusing though, @hrwakeford do you have any ideas on how to make that last bit a little clearer? This would probably be best in not a notebook, but since we didn't set up any documentation yet apart from the readme and this tutorial, it doesn't really have anywhere else to live and I do think it is useful to have it somewhere, albeit maybe in a slightly different form.

**Comment by ivalaginja** [comment]
@hrwakeford I fixed some more typos but will leave this to you now if you want to make some changes. I won't touch this until we have consulted over what to do with that last section in the notebook.

**Comment by hrwakeford** [comment]
With regards to the last part of the notebook I think that we should keep it there but make it look a little different. We can deal with that on the next review.

**Comment by ivalaginja** [comment]
Concerning the error in the notebook, can you confirm that this does *not* happen when you run the notebook for the first time after opening it? I can recreate the error when I run the notebook fully once, and then try to use `Cell --> Run all` again.

This is because the very first cell with code uses a relative path to find the repository in the first place. When this cell is run a second time, the relative path puts you in the wrong place, since it's been run twice now. I can fix this by adding a cell to define a function that always finds the correct absolute path. The alternative is to write in bold red letters not to run that cell twice, but I think I prefer the first option.

As for the data, that is an easy fix. However, do we want data that has a relatively obvious global slope in it, like we have it now? I can drop in the data you provided very easily, would you want me to sneak in a linear slope into that? @hrwakeford

**Comment by ivalaginja** [comment]
@hrwakeford I have added a warning to not run the import cell twice as the functions I wanted to use to get the absolute path to the notebook location only gave me errors.

As for the new data in the notebook, do you want me to add a linear slope to the data you posted above (see my comment above)?

**Comment by hrwakeford** [comment]
I have made some changes to the notebook and pushed those. 
The changes include:
- descriptions of limb-darkening to include reference to stellar model not limb-darkening models, and stellar parameters not limb-darkening parameters. (These can stay as referenced in the config file I just want to make sure the documentation is done).
- Changed the data_y array to the new one computed, including a different random_scatter array that adds simple gaussian noise instead of structured noise. This will keep it simpler for the two examples in the notebook. The data_y also now contains a slight slope that is not fit in the first run but is in the second one so the comparative description is accurate. 
- small typos.

File: notebooks/tutorials/1_Intro-tutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,1586 @@
...
+    "\n",
+    "# Create simple light curve\n",
+    "data_y = np.array([0.99908000, 0.99912000, 0.99915999, 0.99919999, 0.99923998, \n",
+    "                   0.99927998, 0.99931997, 0.99936002, 0.99940002, 0.98886081, \n",
+    "                   0.98884753, 0.98885626, 0.98888551, 0.98893542, 0.98900574, \n",
+    "                   0.98909820, 0.98921506, 0.98936069, 1.0006400, 1.0006800, 1.0007200, \n",
+    "                   1.0007600, 1.0008000, 1.0008399, 1.0008800, 1.0009201, 1.0009600])\n",
+    "\n",
+    "# Assume each data point has exactly the same uncertainty (error)\n",
+    "uncertainty = np.array([0.0003] * len(y_original))\n",
```
Comment (ivalaginja) [inline notebooks/tutorials/1_Intro-tutorial.ipynb]: `y_original` is not actually defined anywhere, this needs to be `data_y`

**Comment by ivalaginja** [comment]
@hrwakeford this is good to go now. If you approve the changes, feel free to merge this PR.

**Comment by ivalaginja** [comment]
I also left the outputs in the notebook - this is usually not a good idea since the figures in notebooks are giant, but since this is our only tutorial notebook, we might want to keep the outputs visible without having to run it.

**Pull request merged**